wemake_python_styleguide/visitors/ast/complexity/overuses.py

```diff
@@ -22,7 +22,25 @@
     'visit_Bytes',
 ))
 class StringOveruseVisitor(base.BaseNodeVisitor):
-    """Restricts several string usages."""
+    """
+    Restricts repeated usage of the same string constant.
+
+    NB: Some short strings are ignored, as their use is very common and
+    forcing assignment would not make much sense (i.e. newlines or "").
+    """
+
+    _ignored_string_constants: Set[Union[str, bytes]] = {
```

Comment:
We use `frozenset` for this kind of constants.

Reply:
Okay! Will fix that.